### PR TITLE
feat: integrate django test cases into ci pipeline

### PR DESCRIPTION
**Pull request by wd0517**
This PR will add Django unit test cases based on Django-TiDB to the TiDB CI pipeline, the tests should run after every PR merged into TiDB repo.

related PR: https://github.com/pingcap/tidb-test/pull/2151

### CODE REVIEW

File: pipelines/pingcap/tidb/latest/merged_integration_django_test.groovy [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+                    }
+                }
+            }
+        }
+        stage('Tests') {
+            matrix {
+                axes {
+                    axis {
+                        name 'TEST_PARAMS'
+                        values 'django_test/django-orm-test ./test.sh'
```
Comment (purelind) [inline pipelines/pingcap/tidb/latest/merged_integration_django_test.groovy:91]: Please confirm that the directory`django_test/django-orm-test`  contains a startup script name as `test.sh`.

Comment (wd0517) [reply pipelines/pingcap/tidb/latest/merged_integration_django_test.groovy:91]: confirmed, the `test.sh` exists in `django_test/django-orm-test`

**Review by purelind — APPROVED** [review]
/lgtm

**Review by purelind — APPROVED** [review]

**Pull request merged**